pandas/tests/util/test_assert_series_equal.py

```diff
@@ -162,12 +162,14 @@ def test_series_equal_length_mismatch(check_less_precise):
 
     with pytest.raises(AssertionError, match=msg):
         tm.assert_series_equal(s1, s2, check_less_precise=check_less_precise)
+        print("No Exception")
```

Comment:
Can remove this

Reply:
Yes. This was left over from some early debugging - and I missed it.  Will remove.